### ISSUE TITLE
Q: how to generate pictures like Fig. 6 in the paper

### ISSUE THREAD

**Issue by DavidZha1994** [iCGY96/ARPL#10]

**Comment by iCGY96** [comment]
For feature visualization, both feature and RP are set to dimension 2.

For details of the experiment, see the repo below:
https://github.com/KaiyangZhou/pytorch-center-loss

I hope it can help you.

**Comment by rameshshreyas** [comment]
@iCGY96 @DavidZha1994 Do you train the final fc layer to make the visualizations?
